types/stripe/index.d.ts

```diff
@@ -10594,6 +10594,11 @@ declare namespace Stripe {
              * The identifier of the customer to subscribe.
              */
             customer: string;
+
+            /**
+             * A date in the future at which the subscription will automatically get canceled.
+             */
+            cancel_at?: number | null;
```

Comment:
If you're adding this, can you fix the documentation to state:

```
A timestamp at which the subscription should cancel. If set to a date before the current period ends this will cause a proration if prorate=true.
```

per https://stripe.com/docs/api/subscriptions/create#create_subscription-cancel_at?

And also, might as well add fields for `cancel_at_period_end` and `prorate`.